app-android/src/main/java/com/ustadmobile/port/android/view/PersonDetailFragment.kt

```diff
@@ -198,3 +216,101 @@ class PersonDetailFragment: UstadDetailFragment<PersonWithPersonParentJoin>(), P
 
 }
 
+@Composable
```

Comment:
We are not using this - everything is in PersonDetailFragment2. I think this Composable should be deleted.